src/FileConverter.ts

```diff
@@ -199,9 +199,13 @@ export class FileConverter {
         const fileOutputPath: string = output;
         const errorData: string[] = [];
         const converted: boolean = await new Promise<boolean>((resolve) => {
-            const proc: ChildProcess = spawn(multiToolPath,
-                ["transform", doc.uri.fsPath, "-o", fileOutputPath, "-p", "-f"],
-            );
+            const launchMultiToolPath: string = multiToolPath;
```

Comment:
Why this practice of introducing an explaining variable for another variable (that seems to me at least) is sufficiently explained?

Reply:
I can revert this... I was trying to debug the issue and wanted to see the individual steps. No reason for this change.

Reply:
Reverted